action.test.js

```diff
@@ -7,152 +7,152 @@ const exec = require("@actions/exec");
 
 const mockDataChecksum = '5663389ef1a8ec48af6ca622e66bf0f54ba8f22c127f14cb8a3f429e40868582';
 const mockRelease = {
-    assets: [
-        {
-            id: 1,
-            name: "hc-releases_0.11.3_darwin_amd64.zip"
-        },
-        {
-            id: 2,
-            name: "hc-releases_0.11.3_linux_amd64.zip"
-        },
-        {
-            id: 3,
-            name: "hc-releases_0.11.3_windows_amd64.zip"
-        },
-    ],
-    id: "1",
-    name: "v0.11.3",
+  assets: [
```

Comment:
Apologies for the whitespace change noise. I recently switched backends for linting/testing/etc. in my editor and it looks like the defaults for Javascript had some fun :(